src/syft/core/store/store_interface.py

```diff
@@ -136,3 +137,23 @@ def clear(self) -> None:
         Clears all storage owned by the store.
         """
         raise NotImplementedError
+
```

Comment:
We should drop these functions, as we were following a dict-like syntax.

Reply:
Agreed. These are currently here to satisfy the mypy linter. So we can remove them as soon as we are no longer calling them from code.